### PR TITLE
Handling of attr_* methods

### PR DESCRIPTION
**Pull request by jeremyf**
Treat calls to attr_* methods as method definitions. Also treat
attribute assignment as a method call.

### CODE REVIEW

File: lib/debride.rb [hunk 1]
```diff
@@ -259,6 +259,22 @@ def process_call sexp # :nodoc:
       new_name = sexp[3]
       new_name = new_name.last if Sexp === new_name # when is this NOT the case?
       known[new_name] << klass_name if option[:rails]
+    when :attr_accessor then
+      sexp[3..-1].each do |new_name|
```
Comment (zenspider) [inline lib/debride.rb]: These aren't warning free. Are you running the tests with rake? It should be outputting warnings.

**Comment by jeremyf** [comment]
@zenspider - Updated. I was not running tests via rake. It required having hoe installed as a test dependency.

As a matter of curiosity, your gems are of a shape different from others. They don't have a gemspec. If you have a post related to the reasons, I'd love to understand. From the outside looking in, you like to have the most bare bones of gems and their associated files as possible.

**Comment by jeremyf** [comment]
Let me know if I should squash this into a single commit.

**Comment by zenspider** [comment]
Done! Thanks!

**Pull request closed without merging**